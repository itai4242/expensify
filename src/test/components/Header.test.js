import React from 'react'
import {shallow} from 'enzyme'
//import ReactShallowRenderer from 'react-test-renderer/shallow';
// import toJSON from 'enzyme-to-json'
import Header from '../../components/Header';

test('should render header correctly', () => {
    const warrper = shallow(<Header />)
    expect(warrper).toMatchSnapshot();
    // const renderer = new ReactShallowRenderer();
    // renderer.render(<Header />);
    // expect(renderer.getRenderOutput()).toMatchSnapshot();
})