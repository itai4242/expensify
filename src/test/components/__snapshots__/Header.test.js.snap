// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header correctly 1`] = `
<header>
  <h1>
    expensify
  </h1>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    exact={true}
    to="/"
  >
    Dashboard
  </NavLink>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    to="/create"
  >
    Creat Page
  </NavLink>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    to="/help"
  >
    Get Help
  </NavLink>
</header>
`;
